py/torch_tensorrt/_compile.py

```diff
@@ -156,13 +154,13 @@ def compile(
             dynamic_batch=False,
             **kwargs,
         )
-    elif target_ir == _IRType.dynamo_compile:
+    elif target_ir == _IRType.dynamo or target_ir == _IRType.torch_compile:
         return torch_tensorrt.dynamo.compile(
```

Comment:
torch_trt.dynamo.compile should only handle export (i.e no ir flag). Directly call torch.compile from here for ir=“torch_compile”

Reply:
done